client/extensions/woocommerce/app/settings/email/email-settings/components/internal-notification.js

```diff
@@ -19,14 +19,7 @@ import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import FormTextValidation from 'components/forms/form-input-validation';
 import { checkEmails } from './helpers';
 
-const InternalNotification = ( {
-	item,
-	recipient,
-	checked,
-	onChange,
-	isPlaceholder,
-	placeholder,
-} ) => {
+const InternalNotification = ( { item, recipient, checked, onChange, loading, translate } ) => {
```

Comment:
Like with customer notifications, it looks like you're more interested in ! loading than in loading, so please consider having loaded as the prop instead for clarity/brevity's sake

Also please add whichever you choose to the propTypes for this component.